src/components/ha-dialog.ts

```diff
@@ -127,6 +127,7 @@ export class HaDialog extends DialogBase {
         border-radius: var(--ha-dialog-border-radius, 28px);
         -webkit-backdrop-filter: var(--ha-dialog-surface-backdrop-filter, none);
         backdrop-filter: var(--ha-dialog-surface-backdrop-filter, none);
+        background: var(--ha-dialog-surface-background);
```

Comment:
You are still overriding the background value now, if the variable is not set it will not have a background.
```suggestion
        background: var(--ha-dialog-surface-background, var(--mdc-theme-surface, #fff));
```

Reply:
You are right, without this it'd fall back to `rgba(0,0,0,0)`